core/sis-feature/src/main/java/org/apache/sis/filter/LeafExpression.java

```diff
@@ -248,13 +159,38 @@ public PropertyTypeBuilder expectedType(FeatureType ignored, final FeatureTypeBu
          * Invoked when a new attribute type need to be created for the given standard type.
          * The given standard type should be a GeoAPI interface, not the implementation class.
          */
-        private static <T> AttributeType<T> newType(final Class<T> standardType) {
+        private static <R> AttributeType<R> newType(final Class<R> standardType) {
             return createType(standardType, Names.createLocalName(null, null, "Literal"));
         }
+    }
+
+
+
+
```

Comment:
Unnecessary extra lines? No harm in leaving them though :man_shrugging:

Reply:
A single line is used between methods, but in this case it is a separation between bigger entities (inner classes). I have no strong opinion about what should be the formatting guideline in those cases however. If there is a well-established practice I would be happy to apply.

Reply:
No strong opinion either, just pointed in case it was unintentional :-)